calculator.rb

```diff
@@ -0,0 +1,82 @@
+# Semret Nicodimos
+# Calculator project
+# 8/7/2018
+
+# Welcome message to the user
+
+puts "Welcome to the Calculator Program! Which mathematical operations would you like to practice today?"
+puts "1. add(+)
+2. subtract(-)
+3. multiply(*)
+4. divide(/)"
+puts "Please choose one operation(name or symbol): "
+operation = gets.chomp.to_s.downcase
+
+# defining methods for math operations
+# additions
+def addition(num1, num2)
+  return num1 + num2
+end
+
+# subtractions
+def subtraction(num1, num2)
+  return num1 - num2
+end
+
+# multiplication
+def multiplication(num1, num2)
+  return num1 * num2
+end
+
+# dividsion
+def division(num1, num2)
+  return num1 / num2
+end
+
+# Loop to ask for and check input for operations
+attempt = 0
+while !(operation == "add" || operation == "+" || operation == "subtract" || operation == "-" || operation == "multiply" || operation == "*" || operation == "divide" || operation == "/")
+  if attempt < 3
+    puts "Oops your input was not recognized. Please try again "
```

Comment:
This part where you get an operation from the user is another place where it would be good for readability to wrap it in a method.